File: src/main/java/ai/aimachineserver/config/AppConfig.java
package ai.aimachineserver.config;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;

@Configuration
public class AppConfig {
    @Value("${custom.db-user-admin-username}}")
    public String dbUserAdminUsername;

    @Value("${custom.db-user-admin-password}")
    public String dbUserAdminPassword;

    @Value("${custom.ai-service-url}")
    public String aiServiceUrl;
}

File: src/main/kotlin/ai/aimachineserver/config/AppConfig.kt
package ai.aimachineserver.config

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration

@Configuration
class AppConfig {
    @Value("\${custom.db-user-admin-username}")
    lateinit var dbUserAdminUsername: String

    @Value("\${custom.db-user-admin-password}")
    lateinit var dbUserAdminPassword: String

    @Value("\${custom.ai-service-url}")
    lateinit var aiServiceUrl: String
}
